backend/app/models/match.rb

```diff
@@ -1,2 +1,4 @@
 class Match < ApplicationRecord
+    validates :user1_id, allow_blank: false
+    validates :game_configuration_id, allow_blank: false
```

Comment:
I think that this should also be forced to be non-negative.

Reply:
Fixed